docs/source/user/pycc.rst

```diff
@@ -38,6 +38,9 @@ Limitations
 #. Each exported function can have only one signature (but you can export
    several different signatures under different names).
 
+#. Exported functions do not check the types of the arguments that are passed
+   to them; the callee is expected to provide arguments of the correct type.
```

Comment:
```suggestion
   to them; the caller is expected to provide arguments of the correct type.
```
is it not that the caller of the AOT function has to ensure the data is of suitable type for use as arguments?

Reply:
Doh!